terraform/a2ha-terraform/modules/s3/templates/createS3Bucket.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo $1
+echo $2
```

Comment:
what is $2 ?

Reply:
$2 is a aws_region variable used for the flag`--create-bucket-configuration LocationConstraint=$2`